httpx/_utils.py

```diff
@@ -23,7 +23,7 @@
     {chr(c): "%{:02X}".format(c) for c in range(0x1F + 1) if c != 0x1B}
 )
 _HTML5_FORM_ENCODING_RE = re.compile(
-    r"|".join([re.escape(c) for c in _HTML5_FORM_ENCODING_REPLACEMENTS.keys()])
+    r"|".join([re.escape(c) for c in _HTML5_FORM_ENCODING_REPLACEMENTS])
```

Comment:
I could go either way on this one. Possibly it's more obvious with `.keys()` what the effect is. Possibly not. But, okay, sure.